libs/Microsoft.MixedReality.WebRTC.Native/src/peer_connection.h

```diff
@@ -508,4 +508,36 @@ class PeerConnection : public webrtc::PeerConnectionObserver,
   PeerConnection& operator=(const PeerConnection&) = delete;
 };
 
+class AudioReadStream {
+ public:
+  AudioReadStream(PeerConnection*, int bufferMs);
+
+  int Read(int sampleRate, float data[], int dataLen, int numChannels) noexcept;
```

Comment:
+docs (in particular : what is the `int` return value?)

Reply:
TBD - some kind of status such as: 
* normal read (success)
* underflow (audio didn't arrive from webrtc in time, )
* overflow (reads happening too slowly)

Reply:
I'll remove the return value for now. I think it may be useful to revisit and add a way for the caller to get a status somewhat like below. Perhaps with millisecond values instead of number of samples. This would let the app show some UI if the connection was going bad in some way. 

struct Status {
 int samplesRead;
 int samplesDropped;
 int samplesMissing;
}